core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/handlers/CopyTextHandler.java

```diff
@@ -53,7 +53,6 @@ public String getSelectionAsText() {
       } else {
         builder.append(o.toString());
       }
```

Comment:
when there is several elements on the selection, we still need a separator between them. otherwise it will be weirdly displayed

Reply:
Ok, added a white space as separator.